src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@ public interface MachineService {
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<Machine> getAll(Class<? extends Machine> type);
```

Comment:
Don't use raw types. Always parametrize interfaces and classes if it is possible. You should parametrize the return type of `getAll` method according to the type parameter `T` of the `MachineService` interface. For example: `List<T> getAll(Class<? extends T> type);`